tests/unit/Cassandra/TimeTest.php

```diff
@@ -23,6 +23,10 @@
  */
 class TimeTest extends \PHPUnit_Framework_TestCase
 {
+    public function setUp() {
+        date_default_timezone_set("UTC");
```

Comment:
Not related and can be removed from this PR; was just an issue from time to time while running the unit tests in Windows when the timezone was not configured properly.